src/main/java/com/uber/cadence/client/WorkflowClient.java

```diff
@@ -131,7 +131,7 @@ static WorkflowClient newInstance(IWorkflowService service, WorkflowClientOption
 
   WorkflowClientOptions getOptions();
 
-  IWorkflowService getService();
+  static IWorkflowService getService();
```

Comment:
this seems like an "almost definitely not correct" change tbh.  this is an extremely large change - is it actually intended?